Dockerfile

```diff
@@ -13,18 +13,16 @@
 # limitations under the License.
 
 # golang:1.19
-FROM golang@sha256:25de7b6b28219279a409961158c547aadd0960cf2dcbc533780224afa1157fd4 AS base
+FROM golang@sha256:25de7b6b28219279a409961158c547aadd0960cf2dcbc533780224afa1157fd4 AS build
 WORKDIR /src
 ENV CGO_ENABLED=0
 COPY go.* ./
 RUN go mod download
 COPY . ./
+RUN make build-scorecard
 
-FROM base AS build
-ARG TARGETOS
-ARG TARGETARCH
-RUN CGO_ENABLED=0 make build-scorecard
-
-FROM gcr.io/distroless/base:nonroot@sha256:99133cb0878bb1f84d1753957c6fd4b84f006f2798535de22ebf7ba170bbf434
+# https://github.com/chainguard-images/images/tree/main/images/static
+# latest
+FROM cgr.dev/chainguard/static@sha256:b25ab001448354017969e5abe42496fb107d759d429e69ef69b7da45f0e6cf0e
```

Comment:
Please share the advantages of each option so we can compare them. Thank you!

Reply:
<img width="1520" alt="Screen Shot 2023-01-11 at 9 32 11 PM" src="https://user-images.githubusercontent.com/16693043/211890958-acfcf9fb-abdf-4d10-bdab-864be9093523.png">
<img width="765" alt="Screen Shot 2023-01-11 at 9 30 09 PM" src="https://user-images.githubusercontent.com/16693043/211890962-eedcd4d2-7dd2-449a-9782-d5eb7507800f.png">

Reply:
@developer-guy  Thanks for the update!

 @laurentsimon @spencerschrock  I am OK with this change. Thoughts?

Reply:
Skimmed https://edu.chainguard.dev/chainguard/chainguard-images/faq/ and https://github.com/chainguard-images/images/tree/main/images/static:

> The main difference is in the implementation. The Google distroless images are built with [Bazel](https://bazel.build/) and based on the Debian distribution, whereas Chainguard Images are built with [apko](https://edu.chainguard.dev/open-source/apko) based on the [Wolfi](https://edu.chainguard.dev/open-source/wolfi) or Alpine distributions.

Just to add to the attestation differences mentioned above.

> The image has a single user nonroot with uid 65532, belonging to gid 65532.

This should be fine here for this scorecard image. The scorecard-action however needs root (which is another Dockerfile in another repo, so that would be an adoption blocker there.

Note: there are other Dockerfiles in this repo, that would be good to have everything on the same page eventually. It might make sense to update the cron images after that pipeline is restored. There's also the attestor images, @raghavkaul do you see this change impacting that?

Reply:
I would recommend that we upgrade one of them and then move it across to cron.

Reply:
>there are other Dockerfiles in this repo, that would be good to have everything on the same page eventually. 

Yep, this is the reason why I marked that issue as `WIP` 😇